API/Services/Tasks/Scanner/ProcessSeries.cs

```diff
@@ -818,6 +818,7 @@ private void AddOrUpdateFileForChapter(Chapter chapter, ParserInfo info, bool fo
             var file = new MangaFileBuilder(info.FullFilePath, info.Format, _readingItemService.GetNumberOfPages(info.FullFilePath, info.Format))
                 .WithExtension(fileInfo.Extension)
                 .WithBytes(fileInfo.Length)
+                .WithHash()
```

Comment:
I'd love to know the overhead of hashing files, perhaps a benchmark can help us understand. (See the Benchmark project)

Reply:
I set up the benchmark and will be committing it soon. I set up the benchmark to build a MangaFile using the MangaFileBuilder. I went ahead and ran the benchmark on hashing a 300 KB epub. Here are the results:
```
| Method                  | Mean      | Error     | StdDev    | Median    | Ratio | RatioSD | Rank | Allocated | Alloc Ratio |
|------------------------ |----------:|----------:|----------:|----------:|------:|--------:|-----:|----------:|------------:|
| TestBuildManga_baseline |  40.18 us |  1.631 us |  4.731 us |  38.49 us |  1.00 |    0.00 |    1 |     922 B |        1.00 |
| TestBuildManga_withHash | 230.09 us | 10.654 us | 31.415 us | 229.86 us |  5.81 |    0.99 |    2 |    7151 B |        7.76 |
```

Reply:
One thing to keep in mind about the hashing algorithm is that a 300KB file will only have 4KB of its contents hashed. At most, the algorithm will hash 10 KB of the file. That means that even though there are files much larger than 300KB, they shouldn't take that much longer to hash.

Reply:
For Koreader, is this only for epub files? Because if so, I think we could swallow this type of overhead. If it's for archives as well, then we might want to look into offloading this into a separate task that performs the hashing. This will offload the extra spend on the scan loop and shortly make it available after scan for syncing.

Reply:
I switched it to only hash EPUBs. I don't think there is any sense in hashing the other files as I don't believe Koreader supports Archives. It supports PDFs though. We can add support for that later.